presto-main/src/main/java/com/facebook/presto/type/DecimalOperators.java

```diff
@@ -83,58 +86,57 @@ private static SqlScalarFunction decimalAddOperator()
                 .signature(signature)
                 .implementation(b -> b
                         .methods("addShortShortShort")
-                        .withExtraParameters(DecimalOperators::shortRescaleExtraParameters)
+                        .withExtraParameters(DecimalOperators::extractScaleParameters)
                 )
                 .implementation(b -> b
                         .methods("addShortShortLong", "addLongLongLong", "addShortLongLong", "addLongShortLong")
-                        .withExtraParameters(DecimalOperators::longRescaleExtraParameters)
+                        .withExtraParameters(DecimalOperators::extractScaleParameters)
                 )
                 .build();
     }
 
     @UsedByGeneratedCode
-    public static long addShortShortShort(long a, long b, long aRescale, long bRescale)
+    public static long addShortShortShort(long a, long b, long aScale, long bScale)
```

Comment:
no, I assumed you have tested it so there is no point in rebenchmarking it for every operator